### PR TITLE
Agenda search results fix

### PR DESCRIPTION
**Pull request by thecalcc**
NHUB-452

## Checklist

- [ ] This pull request is not adding new forms that use redux
- [ ] This pull request is adding missing TypeScript types to modified code segments where it's easy to do so with confidence
- [ ] This pull request is replacing `lodash.get` with optional chaining for modified code segments

### CODE REVIEW

File: assets/search/actions.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ import {
 } from './selectors';
 
 import {context} from 'selectors';
+import {fetchItems} from 'agenda/actions';
```
Comment (MarkLark86) [inline assets/search/actions.ts:25]: Does this now not work in The Wire section? Originally it was calling the `wire/actions/fetchItems` and now its calling the `agenda/actions/fetchItems`. Both those 2 actions do different things.

We would need a better solution here that works for all sections, using dispatching `search/actions/toggleFilter` and `search/actions/setCreatedFilter`

Comment (MarkLark86) [reply assets/search/actions.ts:25]: Maybe we should:
* remove the `fetchItems` from the `toggleFilter` & `setCreatedFilter` dispatch actions
* Add a call to `fetchItems` from where `toggleFilter` & `setCreatedFilter` actions are executed (filter panel etc)

**Review by MarkLark86 — CHANGES_REQUESTED** [review]

File: assets/search/actions.ts (hunk 1 above)
Comment (thecalcc) [reply assets/search/actions.ts:25]: I checked wire and it's working but yes, I think we need a better solution to clearly define where the search is coming from, I'm just still thinking about it and I'll discuss it with Tomas

Comment (MarkLark86) [reply assets/search/actions.ts:25]: I believe this needs to be fixed sooner rather than later, so we cannot wait for Tomas

Comment (MarkLark86) [reply assets/search/actions.ts:25]: I'm not convinced this is the solution, as the `agenda/actions/fetchItems` runs a query against the Agenda API. So, when a filter is removed from the search results bar, a search against Agenda would happen even in the Wire page

**Pull request merged**